frontend/src/components/common/modal/modal.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'common/types/types';
+import { Button } from 'components/common/common';
+import { useState } from 'hooks/hooks';
+import { ReactNode } from 'react';
```

Comment:
Do we import elements from library first and then from other directories? (Idea about putting import {ReactNode} on the first line of file), do not we? @andanfe

Reply:
Yes, you're right
But seems like `eslint-plugin-simple-import-sort` doesn't think so and sorts imports without relative path just in alphabetic order as one group
If it possible we will search for better configuration of this plugin, but for now it is what it is :c